docs/en/trust_marks.rst

```diff
@@ -0,0 +1,245 @@
+
+ 2. **Dynamic** Validation. The Federation members can query the endpoint :ref:`trust mark status<federation_endpoint>` supplied by its issuer (claim **iss**), for a real-time checking of the TMs that it has issued.
+
+All the entities that release Trust Marks, MUST expose a Trust Mark status endpoint for allowing the 
+**dynamic** validation.
+
+.. seealso:: 
+
+  - `OIDC-FED#Section.5.3.2`_
+
+
+Trust Mark Revocation
+---------------------
+
+A Trust Mark can be revoked at any moment only and exclusively by the issuing subject. 
+For example, in case of exclusion of an Aggregated Subject by the Federation Authority, it communicates the exclusion of the Aggregated Subject to the SA. Consequently, the SA MUST revoke the TM for its subordinate.
+
+
+.. note::
+   
+   In case of TM revocation, the **dynamic** validation gives a negative result, while the **static** 
+   validation keeps on giving a positive result, unless the signing encryption keys of the TM-releasing subject are rotated.
+
+
+.. _ComposizioneTM:
+
+Trust Mark Composition
+----------------------
+
+The claims defined inside the TMs, comply to what defined in the OIDC Federation 1.0 (`OIDC-FED`_) standard. Following, the list.
+
+.. list-table::
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+      - **Supported by**
+    * - **iss**
+      - String. URL that uniquely identifies its issuing Authority.
+      - |spid-icon| |cieid-icon|
+    * - **sub**
+      - String. URL that uniquely identifies the subject for which the Trust Mark has been issued.
+      - |spid-icon| |cieid-icon|
+    * - **id**
+      - String. Unique identifier of the Trust Mark. It is an URL with the following structure: |br|
+        **<TA domain>/<entity_type>/<trustmark_profile>/** |br|
+        non-normative example: ``https://registry.interno.gov.it/openid_relying_party/public/``
+      - |spid-icon| |cieid-icon|
+    * - **iat**
+      - UNIX Timestamp with the JWT generation time, coded as NumericDate as indicated at :rfc:`7519`
+      - |spid-icon| |cieid-icon|
+    * - **logo_uri**
+      - String. An URL that points to the logo that represents the Trust Mark.
+      - |spid-icon| |cieid-icon|
+    * - **exp**
+      - UNIX Timestamp with the JWT expiry time, coded as NumericDate as indicated at :rfc:`7519`
+      - |spid-icon| |cieid-icon|
+    * - **ref**
+      - String. URL that points to information present on the web, about this Trust Mark
+      - |spid-icon| |cieid-icon|
+    * - **organization_type**
+      - String. Specifies if the entity belongs to the Italian Public Administration or the private sector (**public** or **private**)
+      - |spid-icon| |cieid-icon|
+    * - **id_code**
+      - String. Identification code of the Organization. Depending on the organization type, it must be
```

Comment:
Organization is uppercased or not? IS this a defined term? if yes, uppercase the next def